core/modules/parser/QSMySqlListener.cc

```diff
@@ -90,13 +107,13 @@ void QSMySqlListener::exit##NAME(QSMySqlParser::NAME##Context * ctx) {\
 } \
 
 
-#define CHECK_EXECUTION_CONDITION(CONDITION, MESSAGE) \
+#define CHECK_EXECUTION_CONDITION(CONDITION, MESSAGE, CTX) \
```

Comment:
I would suggest renaming this macro into what it actually does:
ASSERT_EXECUTION_CONDITION

Reply:
naming is the hardest part! good suggestion.